requirements.txt

```diff
@@ -1 +1,2 @@
 discord.py==2.2.2
+sqlalchemy==2.0.5.post1
```

Comment:
this change doesn't belong here I think :slightly_smiling_face:

Reply:
Yes indeed, I fixed it

Reply:
not really :upside_down_face: